Dockerfile

```diff
@@ -24,33 +27,48 @@ LABEL maintainer="Microsoft" \
 
 WORKDIR azure-cli
 COPY . /azure-cli
-# pip wheel - required for CLI packaging
-# jmespath-terminal - we include jpterm as a useful tool
-RUN pip install --no-cache-dir --upgrade pip wheel jmespath-terminal
+
 # bash gcc make openssl-dev libffi-dev musl-dev - dependencies required for CLI
-# jq - we include jq as a useful tool
 # openssh - included for ssh-keygen
 # ca-certificates
 # wget - required for installing jp
-RUN apk add --no-cache bash gcc make openssl-dev libffi-dev musl-dev jq openssh \
-    ca-certificates wget openssl git && update-ca-certificates
-# We also, install jp
-RUN wget https://github.com/jmespath/jp/releases/download/0.1.2/jp-linux-amd64 -qO /usr/local/bin/jp && chmod +x /usr/local/bin/jp
-
+# jq - we include jq as a useful tool
+RUN apk add --no-cache bash openssh ca-certificates jq wget openssl git \
```

Comment:
Since the essence of this change is to combine all the installation commands in one RUN so as to reduce the number of layers, it would be nice to also gather all the comments and put them ahead of the RUN command instead of spreading them between the lines.

Reply:
Surely it is way more useful to see comments in close proximity to the code it is commenting on?

I would rather remove the comments then move them into one large uber comment.

Reply:
I think you can remove the comments.